src/query/index.ts

```diff
@@ -98,14 +105,17 @@ const filterToApiQueryFilter = (_filters: QueryStoreFilter[], timezone = 'UTC')
 export class QueryHelper {
     private static timezone: ComputedRef<string> | undefined;
 
+    private static referenceStore: ComputedRef<Object> | undefined;
+
     private _keyMap: Record<string, KeyItem> = {};
 
     private _filters: QueryStoreFilter[] = [];
 
     private _orFilters: QueryStoreFilter[] = [];
 
-    static init(timezone: ComputedRef<string>) {
+    static init(timezone: ComputedRef<string>, referenceStore: ComputedRef<Object>) {
         QueryHelper.timezone = timezone;
+        QueryHelper.referenceStore = referenceStore ?? undefined;
```

Comment:
```suggestion
    static init(timezone: ComputedRef<string>, referenceStore?: ComputedRef<Object>) {
        QueryHelper.timezone = timezone;
        QueryHelper.referenceStore = referenceStore;
```

referenceStore가 undefined일 수도 있는데 required parameter인 것은 논리적인 모순이므로, 위처럼 바꾸는게 좋을 것 같습니다!